test/sanity_test.go

```diff
@@ -56,12 +56,21 @@ func TestMyDriver(t *testing.T) {
 		StringData: map[string]string{
 			"remote":     "my-s3",
 			"remotePath": "giab/",
+			"secretKey":  "cw_0x689RpI-jtRR7oE8h_eQsKImvJapLeSbXpwF4e4=",
 			"configData": `[my-s3]
-type=s3
+type=<sensitive>
 provider=AWS`},
 		Type: "Opaque",
 	}, metav1.CreateOptions{})
 
+	// create secret containing saved storage secrets. `type` which is `s3` is encrypted like a secret
+	// if decryption fails, then the storage cannot be mounted
```

Comment:
should we duplicate this test to test once with encrypted secrets and once the original test, to ensure that everything still works if there's no encrypted secret?